scripts/run.sh

```diff
@@ -23,19 +23,74 @@ then
     WEBVERSION=$4
 fi
 
-OS="lin"
```

Comment:
We cannot get rid of OS detection. The setup program uses this for things

Reply:
It's defined some lines below 👍 
`OS="nix"`

OS is checked by `/util/Setup/Program.cs`, and waits for `win` or anything else :)

Reply:
OS must be `win`, `lin`, or `mac`. There is more than just "win or anything else" detection in the setup program.

Reply:
OK, just done !